fares-data-build-tool/create-data
# Feat: Include check for BODS services after search

## Description

This ticket checks if bods has services on searchOperators page

## Testing instructions

- Select Multi-operator fare,  then on page /searchOperators
- Select some bods with no service it should return error with name of the Bods with no services.
